services/api/.env

```diff
@@ -17,7 +17,8 @@ JWT_SECRET=m6bv7!m|E:7:*X3Mx7I)2+Y>B70+&K
 
 # Postmark mail delivery
 POSTMARK_FROM=no-reply@bedrock.foundation
-POSTMARK_APIKEY=
+POSTMARK_API_KEY=
+POSTMARK_DEV_TO=
```

Comment:
Just a shortcut so that you don't have to do that. It's quite common to debug emails. This does apply to postmark too because it only applies to `Postmark.ServerClient` ... we don't really have any other mailer

Reply:
I would say this is business logic of mailer and not postmark ... we calling the wrapper "mailer" because we are not bound to postmark ... can we easy be swapped ... recall AD with blaze or what ever it was called.

Reply:
Then why is it `POSTMARK_FROM` and not `MAILER_FROM`? I'm fine with being accurate but you have to weight it against path of least surprise where the question will be "so what's the difference between "postmark" and "mailer"?" when in fact there is none